UPGRADING.rst

```diff
@@ -20,6 +20,17 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#5280 Enable FIPS mode on GitLab instance
+=========================================
+
+Operator
+~~~~~~~~
+
+Manually deploy the ``gitlab`` component of any main deployment, except
+``prod``, just before pushing the merge commit to the GitLab instance in that
```

Comment:
This appears to contradict the operator checklist item to deploy to `prod.gitlab` for the promotion PR.

Reply:
That was an oversight, thanks for catching it.